ci/set-pipelines.sh

```diff
@@ -22,26 +22,34 @@ readonly FLY_TARGET="spring-credhub"
 readonly GITHUB_REPO="https://github.com/spring-projects/spring-credhub"
 
 set_pipeline() {
-	local pipeline_name pipeline_definition branch ci_image_tag
+	local pipeline_name pipeline_definition branch version
```

Comment:
In other projects, I've come to dislike `version` because it can be pretty ambiguous. In some places we've used `release_series` for vars that have values such as `3.1.x` etc... wdyt?

Reply:
I like that!